package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "laboperator-middleware-development",
+  "version": "1.0.0",
+  "description": "Laboperator Middleware Development Config & Tools",
+  "repository": "git@github.com:labforward/laboperator-middleware-development.git",
+  "author": "labforward GmbH",
```

Comment:
```suggestion
  "author": "Labforward GmbH",
```

Reply:
I thought the company legal name is with lowercase `l`?

Reply:
Nope